### PR TITLE
AAP-25510 Create safe plugin variable for Red Hat Insights EDA plugin and others

### PR DESCRIPTION
**Pull request by jself-sudoku**
Added a new procedure:

- downstream/titles/aap-installation-guide/platform/platform/proc-add-eda-safe-plugin-var.adoc

### CODE REVIEW

**Review by jself-sudoku — COMMENTED** [review]
@jamesmarshall24 - can you provide approval on this?

**Review by jamesmarshall24 — APPROVED** [review]

File: downstream/modules/platform/proc-add-eda-safe-plugin-var.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+When using plugins such as redhat.insights_eda or other similar plugins to run rulebook activations in {EDAcontroller}, you must add a safe plugin variable to a directory in {PlatformNameShort}. This would ensure connecton between {EDAcontroller} and the source plugin and display port mappings correctly. 
+
+.Procedure
+
+. Create a directory for the safe plugin variable: `mkdir -p ./group_vars/automationedacontroller`
+. Create a file within that directory for your new setting (for example, `touch ./group_vars/automationedacontroller/custom.yml`)
+. Add the `automationedacontroller_safe_plugins` variable along with a comma-separated list of plugins to enable for {EDAcontroller}. For example: 
++
+----
+automationedacontroller_safe_plugins: “ansible.eda.webhook, ansible.eda.alertmanager”
```
[the remark this answers is not in the document]
Comment (jamesmarshall24) [reply downstream/modules/platform/proc-add-eda-safe-plugin-var.adoc:15]: So this is going to be backported for 2.4 and then updated for 2.5. For that, I'll approve.

**Review by jamesmarshall24 — APPROVED** [review]

File: downstream/modules/platform/proc-add-eda-safe-plugin-var.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (jself-sudoku) [reply downstream/modules/platform/proc-add-eda-safe-plugin-var.adoc]: Hi James. I was going to backport this to 2.4 first, then update main, and backport to 2.5. After our discussion, I'll just update the 2.4 branch and ONLY backport the current content to 2.5.

**Comment by michellemacrh** [comment]
LGTM

**Pull request merged**